src/request.rs

```diff
@@ -151,6 +151,80 @@ impl SignableRequest for reqwest::Request {
     }
 }
 
+/// Implement `SignableRequest` for [`http::request::Parts`]
+impl SignableRequest for http::request::Parts {
```

Comment:
I feel like we don't need this since our goal is to remove `SignableRequest`

Reply:
this can be considered as a intermediate transition, we can remove it in the next pr

Reply:
Makes sense.